packages/examples/solana-cosmos-bridge/src/lib.rs

```diff
@@ -198,11 +198,15 @@ pub async fn serve(
         match res {
             Err(e) => {
                 set.abort_all();
-                return Err(anyhow::anyhow!("Task panicked: {e}"));
+                return Err(KolmeError::TaskPanicked {
+                    details: e.to_string(),
+                });
             }
             Ok(Err(e)) => {
                 set.abort_all();
-                return Err(e);
+                return Err(KolmeError::TaskErrored {
```

Comment:
Sorry, I should have been more specific. It's OK to use anyhow in the tests. It's also OK to skip it. However, we shouldn't add error variants to `KolmeError` that only apply within the test suite.